service/gcs/runtime/runc/runc_test.go

```diff
@@ -34,9 +34,11 @@ func getBundlePath() (string, error) {
 	return filepath.Join(cwd, "testbundle"), nil
 }
 
+var createdContainers []runtime.Container
+
```

Comment:
The reason I had the query to find all existing containers to delete them was because I wanted a stronger guarantee of a clean state after a test finishing running. I didn't want to rely on the tests always reporting back the correct containers, if for instance the code was buggy and a create call reported back the wrong container ID, or actually succeeded in creating a container but returned an error anyway. I know my approach wasn't without its issues anyway (it still needs to call into potentially buggy code to get all existing containers in the first place), but I think it still offers a stronger guarantee of cleaning up everything given buggy create code. I think I'd prefer this method over relying on what containers the create calls claim to create, but I don't feel strongly enough about it to be against your method either.